web-app/js/portal/ui/openlayers/layer/AlaWms.js

```diff
@@ -0,0 +1,88 @@
+OpenLayers.Layer.AlaWMS = OpenLayers.Class(OpenLayers.Layer.WMS, {
+
+    initialize: function(name, url, params, options) {
+
+        params.queryable = true;
+        params.ENV = OpenLayers.i18n("ALAOccurrencesStyle");
+        options.projection = new OpenLayers.Projection("EPSG:4326");
+        options.isBaseLayer = false;
+        options.sphericalMercator = true;
+
+        OpenLayers.Layer.WMS.prototype.initialize.apply(this, [name, url, params, options]);
+    },
+
+    isAla: function() {
+        return true;
+    },
+
+    getAlaGetFeatureInfoString: function() {
+        return "http://biocache.ala.org.au/ws/ogc/getFeatureInfo";
```

Comment:
Should this be in config?

Reply:
I'm getting the following error for GetFeatureInfo as well:

     Host for address 'http://biocache.ala.org.au/ws/ogc/getFeatureInfo?lat=-38.716796875&lon=146.4697265625&radius=50&q=genus%3AMacropus&wkt=POLYGON((67.5439453125%20-64.029296875%2C-157.5439453125%20-64.029296875%2C-157.5439453125%2011.029296875%2C67.5439453125%2011.029296875%2C67.5439453125%20-64.029296875))&pageSize=2' not allowed